tasks/extract_filepaths_for_granule/schemas/input.json

```diff
@@ -0,0 +1,244 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "properties": {
+    "input": {
+      "type": "object",
+      "properties": {
+        "granules": {
+          "type": "array",
+          "items": [
+            {
+              "type": "object",
+              "properties": {
+                "granuleId": {
+                  "type": "string"
+                },
+                "dataType": {
+                  "type": "string"
+                },
+                "version": {
+                  "type": "string"
+                },
+                "files": {
+                  "type": "array",
+                  "items": [
+                    {
+                      "type": "object",
+                      "properties": {
+                        "key": {
+                          "type": "string"
+                        },
+                        "bucket": {
+                          "type": "string"
+                        },
+                        "fileName": {
+                          "type": "string"
+                        },
+                        "path": {
+                          "type": "string"
+                        },
+                        "url_path": {
+                          "type": "string"
+                        },
+                        "type": {
+                          "type": "string"
+                        },
+                        "duplicate_found": {
```

Comment:
Can we get a pattern property here to define legal values?

Reply:
not sure what this means? I updated the input.json take a look at that.

Reply:
In post_to_database there's an example of using regex patterns to validate inputs. Recommend using them if you know that a value is going to be, say "True" or "False". Looks like you're fine here now that this property is gone.